marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/runMapping.sjs

```diff
@@ -13,7 +13,7 @@ function runMapping () {
   let instance = result.envelope.instance;
   try {
     assertions.push(
-      test.assertEqual('2019-12-07', fn.string(instance.OrderType.purchaseDate)),
+      test.assertEqual(xs.string(xs.date(`2019-12-07${serverTimezone}`)), fn.string(instance.OrderType.purchaseDate)),
```

Comment:
No need to change, but it's good to make a function for something like this which is duplicated a couple times - e.g. 

    test.assertEqual(makeDate("2019-12-07"), ...)

Where "makeDate" would handle the timezone bit and calling xs.string/xs.date.

Reply:
It's the same value so I used a variable.